stdeb/util.py

```diff
@@ -1340,13 +1344,13 @@ def build_dsc(debinfo,
 
     #    J. debian/source/format
     os.mkdir(os.path.join(debian_dir,'source'))
-    fd = open( os.path.join(debian_dir,'source','format'), mode='w')
-    fd.write('3.0 (quilt)\n')
-    fd.close()
+    with open(os.path.join(debian_dir,'source','format'), mode='w') as fd:
+        fd.write('3.0 (quilt)\n')
 
-    fd = open( os.path.join(debian_dir,'source','options'), mode='w')
-    fd.write('extend-diff-ignore="\.egg-info$"')
-    fd.close()
+    #    K. debian/source/options
+    if extend_diff_ignore is not None:
+        with open(os.path.join(debian_dir,'source','options'), mode='w') as fd:
```

Comment:
Since this overwrites the previously written file from a few lines above maybe both write should be done in an `if` and `else` block so that only one of them is executed.